src/constant/typography.ts

```diff
@@ -0,0 +1,27 @@
+export enum TYPOGRAPHY {
```

Comment:
might need to add more in this in the future, continue on the Font story issue

Reply:
cool, thanks